changelog/5069.trivial.rst

```diff
@@ -0,0 +1 @@
+The code for the short test summary in the terminal was moved to the terminal plugin.
```

Comment:
Thanks 🙇

Reply:
You're welcome - sounds really trivial after all indeed.. ;)

/me needs to appreciate changelogs more / not getting annoyed by having to add them.. at least `[ci skip]` allows to skip Travis.